### PR TITLE
Break words when wrapping values in environment table

### PR DESCRIPTION
**Pull request by spadgett**
Before:

openshift_web_console

After:

openshift_web_console

@jwforres PTAL

### CODE REVIEW

File: assets/app/views/directives/environment.html [hunk 1]
```diff
@@ -3,10 +3,11 @@
     <tbody>
       <tr ng-repeat="env in envVars">
         <td>{{env.name}}</td>
-        <td>
+        <!-- Avoid awkward wrapping of long, unbroken values like PATH. -->
+        <td style="word-break: break-all;">
```
Comment (jwforres) [inline assets/app/views/directives/environment.html]: we have a word-break class that use the "nicest" break for the various browsers

Comment (spadgett) [inline assets/app/views/directives/environment.html]: The mixin wraps at the first `-`. It uses `word-break: break-word;` for Chrome which gives the result as in the first screenshot :(

Comment (jwforres) [inline assets/app/views/directives/environment.html]: ah k :-\

Comment (jwforres) [inline assets/app/views/directives/environment.html]: i thought word-break: break-all wasnt supported in all browsers though

**Comment by spadgett** [comment]
Moved the style to core.less.

File: assets/app/views/directives/environment.html (hunk 1 above)
Comment (spadgett) [inline assets/app/views/directives/environment.html]: `break-all` seems to be supported:

https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Maybe `break-word` was the problem?

Comment (jwforres) [inline assets/app/views/directives/environment.html]: yeah looks like IE didnt support it till Edge http://www.w3schools.com/cssref/css3_pr_word-break.asp

Comment (jwforres) [inline assets/app/views/directives/environment.html]: yep break-word is what's not supported in IE 11

Comment (spadgett) [inline assets/app/views/directives/environment.html]: Is our mixin using the wrong property? I though it was `word-break: break-all` and `word-wrap: break-word`... Different properties.

https://github.com/openshift/origin/blob/master/assets/app/styles/_mixins.less#L42-L47
https://developer.mozilla.org/en-US/docs/Web/CSS/word-wrap
https://developer.mozilla.org/en-US/docs/Web/CSS/word-break

Comment (jwforres) [inline assets/app/views/directives/environment.html]: bah sorry said the wrong thing, word-break is not supported

Comment (jwforres) [inline assets/app/views/directives/environment.html]: although the doc you linked says something different than what i was
looking at.  so now im not sure

Comment (jwforres) [inline assets/app/views/directives/environment.html]: ok nevermind... there were two IEs in the w3schools list :)  they had edge
listed separately from the old IEs. so yes it does seem to be supported

**Comment by jwforres** [comment]
[merge]

**Comment by spadgett** [comment]
Let me remove the HTML comment now that I moved the style...

**Comment by spadgett** [comment]
Only changed a comment.

[merge]

**Pull request merged**